lib/presenters/form_presenter.rb

```diff
@@ -0,0 +1,26 @@
+require_relative "base_presenter"
+
+class FormPresenter < BasePresenter
+  def initialize(dashboard, resource)
+    @dashboard = dashboard
+    @resource = resource
+  end
+
+  attr_reader :resource
+
+  def attribute_names
```

Comment:
Hm... `attribute_names` is a different interface than the `ShowPresenter` uses...
Not sure if that's worth changing in itself, though.